packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -496,6 +498,36 @@ class ServiceConnectionManager {
   }
 }
 
+class ResolvedUriManager {
```

Comment:
let's move this class to its own file in the service directory: resolved_uri_manager.dart just to be consistent with how the other manager type classes are separated